packages/frontend/components/scaffold-eth/Contract/DisplayVariables.tsx

```diff
@@ -0,0 +1,57 @@
+import { FunctionFragment } from "ethers/lib/utils";
+import React, { useState } from "react";
+import { useContractRead } from "wagmi";
+import ErrorToast from "~~/components/ErrorToast";
+import { tryToDisplay } from "./displayUtils";
+// import { ArrowPathRounded } from "@heroicons/react";
```

Comment:
Getting this error `Cannot find module '@heroicons/react' or its corresponding type`

Reply:
Fixed here 57790daf6d4c3568ed9f149f9af58cc87bf5b182